packages/jobs/lib/processor/handler.ts

```diff
@@ -35,7 +34,14 @@ export async function handler(task: OrchestratorTask): Promise<Result<JsonValue>
         abort(task);
     };
     if (task.isSync()) {
-        return sync(task);
+        const start = Date.now();
+        const res = await sync(task);
+        if (res.isErr()) {
+            metrics.increment(metrics.Types.SYNC_FAILURE);
+        } else {
+            metrics.increment(metrics.Types.SYNC_SUCCESS);
+            metrics.duration(metrics.Types.SYNC_TRACK_RUNTIME, Date.now() - start);
+        }
```

Comment:
It was the best place I found because they can be triggered or run on schedule, please advice if there is a better place. (`processTask` would be better but it's generic and I would have to add some ifs)

Reply:
that's the main entry point for syncs so I think it is ok.

I would prefer to wrap `const res = await sync(task)` in a `execute.sync` span, and automatically get the hits/duration/errors metrics instead of managing metrics individually. 
It also has the main benefit to allow us to add tags that can be helpful for debugging

Reply:
same answer than below, it would give us the volume/duration but not the breakdown per status